plot_cfDNA_scATAC/sc_cfDNA_demo.Rmd

```diff
@@ -0,0 +1,354 @@
+---
+title: "sc_cfDNA_demo"
+output: github_document
+date: '2022-06-04'
+---
+
+```{r setup, include=FALSE}
+
+knitr::opts_chunk$set(echo = TRUE)
+
+# libraries that I like to install
+require("renv")
+require("here")
+require("data.table")
+require("zoo")
+
+# libraries needed for vignette
+require("Matrix")
+require("Signac")
+require("Seurat")
+require("GenomeInfoDb")
+require("BSgenome.Hsapiens.UCSC.hg38")
+require("EnsDb.Hsapiens.v75")
+require("EnsDb.Hsapiens.v86")
+require("ggplot2")
+require("patchwork")
+#require("hdf5r")
+require("biovizBase")
+set.seed(1234)
+
+require(motifmatchr)
+require(JASPAR2020)
+require(TFBSTools)
+
+
+proj_dir = here()
+
+```
+
+## Overview
+
+This file is an augmented version of the scATAC-seq analysis found here: https://satijalab.org/signac/articles/pbmc_vignette.html
```

Comment:
Remind me what the point of this repo is? Are you learning the basics of scATAC-seq or is this to help the undergrads?

Reply:
This is for me to learn and have a reference for how to use Seurat+Signac on cell-free DNA data. So while the vignette focuses on scATAC-Seq data, I have augmented it in small ways to work with cell-free DNA data. I added a blurb of this in https://github.com/greenelab/cfDNA_atac_demo/pull/3/commits/0718bf4ab4e90ec1695652682ec95f955035df9c